cypress/integration/capture.spec.js

```diff
@@ -55,7 +55,7 @@ describe('Event capture', () => {
 
         cy.get('body').click(100, 100).click(98, 102).click(101, 103)
 
-        cy.phCaptures().should('deep.equal', ['$pageview', '$rageclick'])
```

Comment:
Q: Why did this break?

Reply:
You're too fast! I've fixed it now. Essentially, by adding more elements to the playground html, we're rendering this:

<img width="261" alt="Screenshot 2021-04-06 at 13 54 55" src="https://user-images.githubusercontent.com/38760734/113722237-c318fc00-96df-11eb-81af-9f530a920a7d.png">

So clicks a few pixels off were also clicking other elements, and hence that array had more events than just those two.